server/api/archivedOpportunity/archivedOpportunity.routes.js

```diff
@@ -1,22 +1,50 @@
 const mongooseCrudify = require('mongoose-crudify')
 const helpers = require('../../services/helpers')
-const archivedOpportunity = require('./archivedOpportunity')
+const ArchivedOpportunity = require('./archivedOpportunity')
 const { getArchivedOpportunities, getArchivedOpportunity } = require('./archivedOpportunity.controller')
+const { authorizeActions } = require('../../middleware/authorize/authorizeRequest')
+const removeUnauthorizedFields = require('../../services/authorize/removeUnauthorizedFields')
+const { SchemaName } = require('./archivedOpportunity.constants')
+const { Action } = require('../../services/abilities/ability.constants')
+
+const convertRequestToAction = (req) => {
```

Comment:
This looks like a candidate for factoring out - are we defining the same function in all ability files?

Reply:
Yeah I'm getting the impression we could just update the default function that is available and then all of these custom implementations could be removed